.gitattributes

```diff
@@ -0,0 +1 @@
+RELEASE.md merge=union
```

Comment:
come funziona? cosa fa?

Reply:
come suggerito da Piero ieri, porting di quello che si fa su Volto
[4:20 PM](Valutate intanto però di mettere RELEASE.md in .gitattributes in modo che si auto mergino le righe nuove invece di dare conflitto)


[4:20 PM]Come qui https://github.com/plone/volto/blob/master/.gitattributes